### PR TITLE
Fix emoji encoding in URLs for FTP uploads

### PR DESCRIPTION
**Pull request by x04**
Currently if you use FTP/FTPS/SFTP to upload images it ignores the configuration option to ignore emoji's in the file name/URL. That makes it so I can't do things like [https://noyou.gay/🧣🧞👔🧦👵👈💁🚶.png](https://noyou.gay/🧣🧞👔🧦👵👈💁🚶.png), it just comes out to be a terrible URL encoded version of the emojis. With the changes in this PR that is fixed if they choose to ignore emojis.

### CODE REVIEW

**Comment by Jaex** [comment]
I tested it now, it returned this URL:
`https://example.com/test test/test 😀 (🐱).txt`
So it not encodes non emoji parts. This is the reason why I did not add emoji support to FTP too.

**Comment by x04** [comment]
That should fix the issues.

File: ShareX.HelpersLib/Helpers/URLHelpers.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using System.Web;
+using System.Windows.Forms;
```
Comment (JakeSteam) [inline ShareX.HelpersLib/Helpers/URLHelpers.cs:38]: 🤔

Comment (x04) [reply ShareX.HelpersLib/Helpers/URLHelpers.cs:38]: Oh yeah I believe IDE autocompleted that when I was typing something, I'll remove it.

**Comment by x04** [comment]
@Jaex any more input on this?

**Comment by ha6000** [comment]
Bump

**Pull request closed without merging**